DigitalLearningSolutions.Web.Tests/ViewModels/TrackingSystem/Centre/Reports/ReportsViewModelTests.cs

```diff
@@ -18,20 +18,28 @@ public void UsageStatsTableViewModel_reverses_data_in_time()
             {

```

Comment:
I removed all the extra PeriodOfActivity declarations in a previous version of this since they weren't needed for that version of the display strings. If we're adding them back, presumably we need them, but does this mean we should have more assertions in each test?
This applies to all tests in this file.

Also can you capitalise all the // given // when // then for consistency with the rest of the solution.

Reply:
If I understand what you're referring to correctly, they're needed because single, start, end and middle periods are all converted to labels differently, so there need to be additional periods wrapping middle periods to test their conversion. That the start and end periods are converted correctly has its own test, which only has one variant because those conversions don't depend on the period.

Reply:
Ok that's fine then. Wasn't sure that all cases were being tested.